RoddyCore/src/de/dkfz/roddy/config/FilenamePattern.java

```diff
@@ -121,11 +119,13 @@ protected String fillVariables(BaseFile baseFile, String temp) {
     protected String fillVariablesFromSourceFileValues(BaseFile baseFile, String temp) {
         FileStageSettings fs = baseFile.getFileStage();
         String _temp = temp;
-        temp = fs.fillStringContent(temp);
+        if (fs != null)
```

Comment:
I don't think it should ever be null. But in some buggy cases it was. So I decided to have some kind of fallback without wrecking everything and without using try/catch too much.